test/runtests.jl

```diff
@@ -194,4 +194,24 @@ const DATADIR = joinpath(@__DIR__, "data")
 
     @test EDF._size(IOBuffer("memes")) == 5
     @test EDF._size(Base.DevNull()) == -1
+
+    @testset "BDF Files" begin
+        # The corresponding EDF file was exported by 3rd party software based on the BDF,
```

Comment:
can/should you specify the 3rd party software?

Reply:
:zipper_mouth_face: